lib/services/messenger/translators/flowdock.ts

```diff
@@ -103,16 +103,25 @@ export class FlowdockTranslator extends TranslatorScaffold implements Translator
 	 * @returns         Promise that resolve to the thread details.
 	 */
 	private static convertReadConnectionResponse(
-		message: TransmitInformation, response: FlowdockResponse
+		message: TransmitInformation, response: FlowdockMessage[]
 	): Promise<IdentifyThreadResponse> {
-		if (response.length > 0) {
-			const idFinder = new RegExp(`\\[${message.source.service} thread ([\\w\\d-+\\/=]+)]`, 'i');
+		const idFinder = new RegExp(`${message.source.service} thread ([\\w\\d-+\\/=\\_]+)]`, 'i');
```

Comment:
It looks like this regex may have a trailing `]` character, I'm not sure if this is intentional or not

Reply:
Intentional, but not required.